sdk/docs/iot/README.md

```diff
@@ -71,6 +71,30 @@ Feature | Azure SDK for Embedded C | Description
  Retry Policies | &radic;* | The Azure SDK for Embedded C provides guidelines for retries, but actual retries should be handled by the application.
  [IoT Plug and Play](https://docs.microsoft.com/azure/iot-pnp/overview-iot-plug-and-play) | &radic; | IoT Plug and Play enables solution builders to integrate smart devices with their solutions without any manual configuration.
 
+<br>
+
```

Comment:
This shouldn't be necessary. GitHub renders new lines fine. If you were using VSCode and its Markdown Preview, that doesn't handle tables well, and makes it seem like an extra new line is needed, causing the author to think they need to add more new lines like this. That ends up looking odd, due to the extra spacing:
![image](https://user-images.githubusercontent.com/6527137/96674434-a5b4c900-131d-11eb-8fc0-2def648899f5.png)

```suggestion
```